packages/kbn-eslint-import-resolver-kibana/lib/get_webpack_config.js

```diff
@@ -29,6 +29,7 @@ exports.getWebpackConfig = function(kibanaPath, projectRoot, config) {
     // Kibana defaults https://github.com/elastic/kibana/blob/6998f074542e8c7b32955db159d15661aca253d7/src/legacy/ui/ui_bundler_env.js#L30-L36
     ui: fromKibana('src/legacy/ui/public'),
     test_harness: fromKibana('src/test_harness/public'),
+    querystring: 'querystring-browser',
```

Comment:
@spalger Is this acceptable? I think we would prefer not to introduce Webpack aliases. If this is just for the typings, I think we may be able to add a `declare module 'querystring-browser'` definition in `typings/` that rexports the types from Node's `querystring` to `querystring-browser`.